src/coreclr/jit/lsrabuild.cpp

```diff
@@ -3121,6 +3121,11 @@ int LinearScan::BuildOperandUses(GenTree* node, regMaskTP candidates)
     {
         return BuildAddrUses(node, candidates);
     }
+    if (node->OperIs(GT_BSWAP))
+    {
+        BuildOperandUses(node->gtGetOp1(), candidates);
+        return 1;
```

Comment:
Why `return 1`? Presumably we could have a `BSWAP(IND(LEA(base, index))`, where we'd have two registers, and so we should `return BuildOperandUses` instead (or `return BuildIndirUses` for clarity, I suppose).